files/ru/web/api/web_workers_api/functions_and_classes_available_to_workers/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Функции и классы доступные для Web Workers
+slug: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
+translation_of: Web/API/Web_Workers_API/Functions_and_classes_available_to_workers
+---
+
+{{DefaultAPISidebar("Web Workers API") }}
```

Comment:
There was very little left of this file after cleaning up the macro usage, so I synced it with English and converted it to Markdown. I hope that is ok.